graphdepend.ml4

```diff
@@ -88,17 +88,19 @@ module G = struct
 
   module Edges = Set.Make (Edge)
 
-  type t = (Names.GlobRef.t, int) Hashtbl.t * Edges.t
+  module GMap = Map.Make (Globnames.RefOrdered_env)
```

Comment:
Don't we already have this in globnames?

Reply:
Yes and no. The one in globnames doesn't preserve ordering in the fold and iter operations because it uses `HMap` instead of `Map`.